packages/client/test/end-to-end/publish-subscribe.test.ts

```diff
@@ -92,12 +90,26 @@ describe('publish-subscribe', () => {
             })
         }, TIMEOUT * 2)
 
-        it('messages are published encrypted', async () => {
-            await publisherClient.publish(stream.id, PAYLOAD)
-            const messages = await startNetworkNodeAndListenForAtLeastOneMessage(stream.id)
-            expect(messages).toHaveLength(1)
-            expect(messages[0]).toBeInstanceOf(Uint8Array)
-        }, TIMEOUT)
+        describe('NetworkNode receiving', () => {
+            let networkNode: NetworkNode
+
+            beforeEach(async () => {
+                networkNode = await startNetworkNode(stream.id)
+            }, TIMEOUT)
+
+            afterEach(async () => {
+                await networkNode.stop()
+            }, TIMEOUT)
+
+            it('messages are published encrypted', (done) => {
```

Comment:
Could avoid using the done pattern here? Should be possible to implement with async/await